pom.xml

```diff
@@ -135,6 +140,21 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>narayana-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.narayana.jta</groupId>
+        <artifactId>tomcat-jta</artifactId>
+        <version>5.6.3.Final-SNAPSHOT</version>
+      </dependency>
```

Comment:
We really should not be depending on 3rd party SNAPSHOTs.

(We should also move these deps into kie-parent, but that can be done later).